### PR TITLE
Angular-UI-Router

### PR DESCRIPTION
**Pull request by shirishgoyal**

### CODE REVIEW

**Comment by dmorina** [comment]
screen shot 2016-03-02 at 10 29 54 am

File: static/js/crowdsource.routes.js [hunk 1]
```diff
@@ -2,155 +2,323 @@
...
+                views: {
+                    'navbar': navbar,
+                    'content': projectReview
+                },
+                authenticate: true,
                 resolve: {
-                    resolvedData: function ($route, Project) {
-                        return Project.retrieve($route.current.params.projectId);
+                    resolvedData: function ($stateParams, Project) {
+                        return Project.retrieve($stateParams.params.projectId);
```
Comment (dmorina) [inline static/js/crowdsource.routes.js]: i think this is failing

**Comment by dmorina** [comment]
lgtm overall, thanks for this
a comment left above

**Comment by dmorina** [comment]
btw I had a discussion with Christopher some time ago and an idea was to keep 960px for bigger screens and min-width: 768px; so that we have a bit more space

**Comment by shirishgoyal** [comment]
I would rather just use google specs as defined here : https://material-design.storage.googleapis.com/publish/material_v_4/material_ext_publish/0B8olV15J7abPSGFxemFiQVRtb1k/layout_adaptive_breakpoints_01.png

https://www.google.com/design/spec/layout/responsive-ui.html#responsive-ui-breakpoints

**Comment by shirishgoyal** [comment]
Fixed bug in project review

**Comment by dmorina** [comment]
lgtm

**Pull request merged**